### PR TITLE
Add share button with links for each platform

### PR DESCRIPTION
**Pull request by otterbotter**
Trello: https://trello.com/c/StJUxgQF/179-as-a-user-i-would-like-to-share-from-app-to-social-media-and-whatsapp-so-others-in-my-circle-can-see-the-information-and-be-awar

Includes webflow exports:
- https://openup.gitbook.io/cape-agulhas-app/design-system/webflow-exports#2021-04-02-share-button
- Tab Content Fix

@jbothma please confirm if you're happy with the placeholder messages for each sharing platform in `share.js` e.g. the email subject and body, the whatsapp message etc. (they're quite generic like "Have you seen this page? <link>)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -139,11 +185,7 @@ <h1 class="nav-title">Cape Agulhas</h1>
             </a>
           </div>
         </div>
-        <div class="tab-contents__wrap">
-          <div class="tab-content scroll">
-            <div class="grid--default"></div>
```
Comment (otterbotter) [inline src/index.html:144]: I think this has broken the JS- an assertion is failing:

`Assertion failed: console.assert` that asserts that the length of `tab-content` is 1

File: src/js/utils/share.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  const encodedShareMessage = encodeURI(`Have you seen this? ${getPath()}`);
+  return `https://twitter.com/intent/tweet?text=${encodedShareMessage}`;
+}
+
+function getLinkedinShareURL() {
+  return `https://www.linkedin.com/shareArticle?mini=true&url=${getPath()}`;
+}
+
+function getEmailShareURL() {
+  const encodedSubject = encodeURI("Have you seen this page?");
```
Comment (jbothma) [inline src/js/utils/share.js:26]:
```suggestion
  const encodedSubject = encodeURI("Take a look at the Cape Agulhas App");
```

File: src/js/utils/share.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  const encodedBody = encodeURI(`Have a look here: ${getPath()}`);
```
Comment (jbothma) [inline src/js/utils/share.js:27]:
```suggestion
  const encodedBody = encodeURI(`Take a look at the Cape Agulhas App: ${getPath()}`);
```

File: src/js/utils/share.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
+function getPath() {
+  return window.location.href;
+}
+
+function getWhatsappShareURL() {
+  const encodedShareMessage = encodeURI(
+    `I'd like to share this Citizen Engagement page with you: ${getPath()}`
```
Comment (jbothma) [inline src/js/utils/share.js:7]:
```suggestion
    `Take a look at this page in the Cape Agulhas App: ${getPath()}`
```

File: src/js/utils/share.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+  );
+  return `https://wa.me/?text=${encodedShareMessage}`;
+}
+
+function getFacebookShareURL() {
+  return `https://www.facebook.com/sharer/sharer.php?u=${getPath()}`;
+}
+
+function getTwitterShareURL() {
+  const encodedShareMessage = encodeURI(`Have you seen this? ${getPath()}`);
```
Comment (jbothma) [inline src/js/utils/share.js:17]:
```suggestion
  const encodedShareMessage = encodeURI(`Take a look at this page in the Cape Agulhas App ${getPath()}`);
```

**Comment by jbothma** [comment]
Nice! Merge when you've imported the webflow fix

**Pull request merged**